cmd/puma-dev/main_test.go

```diff
@@ -157,6 +157,18 @@ func getURLWithHost(t *testing.T, url string, host string) string {
 	return strings.TrimSpace(string(bodyBytes))
 }
 
+func parseDumpedHTTPHeadersFromBody(body string) map[string]string {
```

Comment:
```suggestion
// see etc/rack-request-headers-dump/config.ru
func parseDumpedHTTPHeadersFromBody(body string) map[string]string {
```

prob worth pointing out why this is here